README.md

```diff
@@ -51,13 +51,13 @@ rootProject {
 ```
 
 ## Configuration
-| Property | Default | Optional? | Description |
+| Property | Default | Required? | Description |
 | --- | --- | --- | --- |
-| automatticProject | null | false | Project that will determine event name
-| username | "anon" | true | Username associated with report |
-| uploadEnabled | true | true | Opt-out flag for sending reports to Tracks |
-| customEventName | null | true | Event name that overrides one set by `automatticProject`, should be used for debug purposes. |
-| debug | false | true | Show additional logs
+| automatticProject | null | yes | Project that will determine event name
+| enabled | null | yes | Enable or disable plugin |
```

Comment:
I've renamed this because now it's not only enablind/disabling `upload` of reports. Also I made this as required as I thought that there's no sense to introduce kind of ambiguity - the plugin should be enabled or disabled only.